config/routes/internal_api.rb

```diff
@@ -32,6 +32,7 @@
     resources :workspaces, only: [:update]
     resources :invoices, only: [:index, :create, :update, :show, :destroy, :edit] do
       post :send_invoice, on: :member
+      get :download, on: :member
```

Comment:
nit-picky suggestion

```suggestion
      member do
        post :send_invoice
        get :download
      end
```